src/NumericalAlgorithms/DiscontinuousGalerkin/MetricIdentityJacobian.hpp

```diff
@@ -0,0 +1,180 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+template <size_t Dim>
+class Mesh;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace dg {
+/*!
+ * \brief Compute the Jacobian determinant times the inverse Jacobian so that
+ * the result is divergence-free.
+ *
+ * The metric identities are given by
+ *
+ * \f{align*}{
+ *
+ * \partial_{\hat{\imath}}\left(J\frac{\partial\xi^{\hat{\imath}}}
+ * {\partial x^i}\right)=0.
+ *
+ * \f}
+ *
+ * We want to compute \f$J\partial\xi^{\hat{\imath}}/\partial x^i\f$ in such a
+ * way that the above metric identity is satisfied numerically/discretely. We
+ * refer to the inverse Jacobian computed this way as the "metric
+ * indentity-satisfying inverse Jacobian".
+ *
+ * The general form is given by:
+ *
+ * \f{align*}{
+ *
+ * 2\left(J\frac{\partial \xi^{\hat{\imath}}}{\partial x^i}\right)_{\breve{s}}
+ *  &=\epsilon_{ijk}
+ *    \sum_{t_{\hat{1}}} \epsilon^{\hat{\imath}\hat{1}\hat{k}}
+ *    D^{(\hat{1})}_{\breve{s}\breve{t}}
+ *    \left(x^j\frac{\partial x^k}{\partial \xi^{\hat{k}}}\right)_{\breve{t}}
+ *    \notag \\
+ *  &+\epsilon_{ijk}
+ *    \sum_{t_{\hat{2}}} \epsilon^{\hat{\imath}\hat{2}\hat{k}}
+ *    D^{(\hat{2})}_{\breve{s}\breve{t}}
+ *    \left(x^j\frac{\partial x^k}{\partial \xi^{\hat{k}}}\right)_{\breve{t}}
+ *    \notag \\
+ *  &+\epsilon_{ijk}
+ *    \sum_{t_{\hat{3}}}\epsilon^{\hat{\imath}\hat{3}\hat{k}}
+ *    D^{(\hat{3})}_{\breve{s}\breve{t}}
+ *    \left(x^j\frac{\partial x^k}{\partial \xi^{\hat{k}}}\right)_{\breve{t}}.
```

Comment:
What are the indices s,t? Gridpoints?

Why are they sometimes under a "breve", but not when under the sum? Can the breve be removed?

Reply:
I use the breves for grid points. I would like to keep them because using `i,j,k`, etc. as spatial indices and as grid points will almost certainly be confusing (I've always found it confusing personally)

Reply:
I find that adding excessive diacritics makes things hard to read (even in rendered form). I thought the convention we were aiming for was "i, j, k" for tensor components and "s, t, ..." for gridpoints? Has this proved insufficient? If that's the case and the breves are necessary, then would you at least agree the index under the sum needs a breve added as well?

Reply:
That's fair. I'll remove the breves (either case I definitely agree that they should match with the sum!) and then explicitly state that `s,t` are over grid points :)

Reply:
Thanks - to me at least that'll be clearer for sure :+1: